examples/send-msg/main.go

```diff
@@ -52,8 +52,17 @@ func newModel() model {
 	}
 }
 
+func doTick() tea.Cmd {
```

Comment:
reusing the function name from the docs - https://pkg.go.dev/github.com/charmbracelet/bubbletea#Tick